config/locales/models/medical_shifts.yml

```diff
@@ -0,0 +1,6 @@
+en:
+  medical_shifts:
+    attributes:
+      shift:
+        nighttime: "Noturno"
```

Comment:
No problem! I'll keep this task of creating the translations for pt-BR.

Here, to follow the standard in English, it should be adjusted to something like this: Nocturnal and Diurnal

What do you think?

Reply:
I'll change to en version, but I prefer to keep nighttime and daytime. I looked for it and I saw usage for these words when the context it's work. Is it ok for you?

Reply:
done! https://github.com/espoo-dev/med_system_backend/pull/203/commits/540c8680ed6ae54e9d052ba7f373ab7822a7ef17